### PR TITLE
Tanium additional log types mapping

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5883

## Description
Added mapping to integrity monitor logs for tanium

### CODE REVIEW

File: Packs/Tanium/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Tanium 
+This pack includes Cortex XSIAM content.
+
+## Configuration on Server Side
+
+### Tanium Integrity Monitor logs
+In order to forward Tanium integrity monitor logs, follow the below steps.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/README.md:7]:
```suggestion
In order to forward Tanium Integrity Monitor logs, follow the steps below.
```

File: Packs/Tanium/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+You will need to configure a Socket Receiver on Tanium side.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/README.md:9]:
```suggestion
You will need to configure a Socket Receiver on the Tanium side.
```

File: Packs/Tanium/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+1. Go to **Modules** > **Interact**.
+2. Copy the below question to the "Ask a Question" box under **Explore Data** and run it.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/README.md:12]:
```suggestion
2. Copy the following question to the "Ask a Question" box under **Explore Data** and run it.
```

File: Packs/Tanium/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+```bash
+Get Computer Name and Last Logged In User and Integrity Monitor - Monitor Events[10,0,360,15,0,""] from all machines
+```
+3. Click **Save** and configure the question as described in the next steps.
+4. Under **Name**, write "XSIAM Integrity Montior".
+5. From the **Content Set** dropdown, select **Base**.
+6. Verify that the text under **Question Text** matches the question mentioned on step 2.
+7. Check the "Reissue this question every" check box, and set it to 2 hours.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/README.md:20]:
```suggestion
7. Check the "Reissue this question every" checkbox, and set it to 2 hours.
```

File: Packs/Tanium/README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+1. Go to **Modules** > **Interact**.
+2. Copy the below question to the "Ask a Question" box under **Explore Data** and run it.
+```bash
+Get Computer Name and Last Logged In User and Integrity Monitor - Monitor Events[10,0,360,15,0,""] from all machines
+```
+3. Click **Save** and configure the question as described in the next steps.
+4. Under **Name**, write "XSIAM Integrity Montior".
+5. From the **Content Set** dropdown, select **Base**.
+6. Verify that the text under **Question Text** matches the question mentioned on step 2.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/README.md:19]:
```suggestion
6. Verify that the text under **Question Text** matches the question mentioned in step 2.
```

File: Packs/Tanium/README.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+8. Go to **Modules** > **Connect**.
+9. Enter a name and description for the connection.
+10. From the **Source** dropdown, select **Saved Question**.
+11. From the **Saved Question Name** dropdown, select **XSIAM Integrity Montior**.
+12. From the **Destination** dropdown, select **Socket Receiver**.
+13. Specify a unique name for the **Destination Name**.
+14. Under **Host**, fill in the name or the IP address of the SIEM.
+15. Specify the port number under **Port**.
+16. Select **JSON** from the dropdown under **Format**.
+17. Check the "Enable Schedule" check box under **Schedule**.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/README.md:30]:
```suggestion
17. Check the "Enable Schedule" checkbox under **Schedule**.
```

File: Packs/Tanium/ReleaseNotes/1_0_28.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+#### Modeling Rules
+##### New: Tanium Modeling Rule
+Added a modeling rule for integrity monitor logs.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/ReleaseNotes/1_0_28.md:3]:
```suggestion
Added a modeling rule for Integrity Monitor logs.
```

File: Packs/Tanium/ReleaseNotes/1_0_28.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+
+#### Parsing Rules
+##### New: Tanium Parsing Rule
+Added a parsing rule for integrity monitor logs to support _time mapping.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/ReleaseNotes/1_0_28.md:7]:
```suggestion
Added a parsing rule for Integrity Monitor logs to support time mapping.
```

**Review by eepstain — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

**Pull request merged**